openquakeplatform_building_class/templates/building-class/building-class.html

```diff
@@ -19,16 +19,15 @@
 {% load i18n %}
 {% load url from future %}
 
-{% block title %} Building Classification Survey {{ block.super }} {% endblock title %}
+{% block title %} Building Classification Survey {% endblock title %}
```

Comment:
Use the new approach.

Reply:
Fixed with this af94bc838a70de0899784043fa4c69b70839bb0e